docs/src/tutorials/applications/optimal_power_flow.jl

```diff
@@ -0,0 +1,545 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Optimal power flow
+
+# *This tutorial was originally contributed by James Foster (@jd-foster).*
+
+# This tutorial formulates and solves an alternating current optimal power flow
+# (AC-OPF) problem, a much-studied nonlinear problem from the field of
+# electrical engineering.
+
+# Once we've formulated and solved the nonlinear problem, we will turn our focus
+# to obtaining a good estimate of the objective value at the global optimum
+# through the use of semidefinite programming.
+
+# The purpose of this tutorial is to highlight JuMP's ability to directly
+# formulate problems involving complex-valued decision variables and complex
+# matrix cones such as the  [`HermitianPSDCone`](@ref) object.
+
+# For another example of modeling with complex decision variables, see the
+# [Quantum state discrimination](@ref) tutorial, and see the
+# [Complex number support](@ref) section of the manual for more details.
+
+# ## Required packages
+
+# This tutorial requires the following packages:
+
+using JuMP
+import Clarabel
+import DataFrames
+import Ipopt
+import LinearAlgebra
+import SparseArrays
+import Test
+
+# ## Initial formulation
+
+# Optimal power flow problems for electrical transmission typically pose the
+# following question: what is the most cost-effective operation of electricity
+# generators while meeting constraints on the safe limits of network components?
+
+# The graph of the network shown here has nine nodes, with three each used for
+# the different purposes of generation ``G`` (nodes 1, 2, and 3), trans-shipment
+# (nodes 4, 6, and 8), and demand ``D`` (nodes 5, 7, and 9).
+
+# ![Nine Nodes](../../assets/case9mod.png)
+
+# We're using the 9-_bus_ network test case `case9mod` to explore this problem.
```

Comment:
```suggestion
```

Paragraph before mentions nine nodes and next sentence says `case9`.

Reply:
Sorry, not sure what you're asking for.

Reply:
I just thought the sentence was superfluous.

Reply:
Ok, will remove the redundancy.